db/data.go

```diff
@@ -190,11 +192,7 @@ func isDuplicateKeyErr(err error) bool {
 		return false
 	}
 
-	if me.Number != errDupEntry {
-		return false
-	}
-
-	return strings.Contains(me.Message, "key 'PRIMARY'")
```

Comment:
Even this is a little too strict. I get `Duplicate entry 'glock/tests/Test_ConcurrentUse' for key 'data.PRIMARY'`

Since there's only one unique constraint on the table, probably sufficient to check the MySQL error code for now.

Reply:
Looks good